index.bs

```diff
@@ -5768,6 +5807,7 @@ type.
         "boolean"
         "byte"
         "octet"
+        "bigint"
```

Comment:
The addition of this new unnamed terminal means the `Other` production needs to be updated to include that terminal here https://github.com/heycam/webidl/blob/c387fed823d2d4ccf8940bbc065f358ccf563073/index.bs#L6998-L6999

(Unnamed/quoted terminals ‘delete the token’ from the set of values represented by a named terminal, so _ident_ is currently inclusive of the string "bigint," but will cease to be, hence the need to explicitly add it to Other, which is inclusive of every terminal except for a few specific punctuation marks.)